File: app/src/main/java/io/mgba/UI/Fragments/Main/FavouritesFragment.java
package io.mgba.UI.Fragments.Main;

import io.mgba.R;

public class FavouritesFragment extends GameFragment {

    @Override
    protected void prepareDrawables() {
        noContentImage.setImageDrawable(getResources().getDrawable(R.drawable.ic_favorite_grey_500_48dp));
        noContentMessage.setText(R.string.no_favourites);
    }
}

File: app/src/main/java/io/mgba/ui/fragments/main/FavouritesFragment.kt
package io.mgba.ui.fragments.main

import androidx.core.content.ContextCompat
import io.mgba.R
import kotlinx.android.synthetic.main.games_list_view.*

class FavouritesFragment : GameFragment() {

    override fun prepareDrawables() {
        noContentIcon.setImageDrawable(ContextCompat.getDrawable(context!!, R.drawable.ic_favorite_grey_500_48dp))
        noContentMessage.setText(R.string.no_favourites)
    }
}
